client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -80,30 +78,30 @@ const PluginMeta = React.createClass( {
 						src={ this.props.plugin.banners.high || this.props.plugin.banners.low } />
 					</div>;
 		}
-	},
+	}
 
 	hasBusinessPlan() {
 		if ( ! this.props.selectedSite ) {
 			return false;
 		}
 		return isBusiness( this.props.selectedSite.plan ) || isEnterprise( this.props.selectedSite.plan );
-	},
+	}
 
-	isWpcomPreinstalled: function() {
+	isWpcomPreinstalled = () => {
 		const installedPlugins = [ 'Jetpack by WordPress.com', 'Akismet', 'VaultPress' ];
 
 		if ( ! this.props.selectedSite ) {
 			return false;
 		}
 
 		return ! this.props.selectedSite.jetpack && includes( installedPlugins, this.props.plugin.name );
-	},
+	};
 
 	renderActions() {
 		if ( ! this.props.selectedSite ) {
 			return (
 				<div className="plugin-meta__actions">
-					<div className="plugin-item__count">
+					<div className="plugin-meta__count">
```

Comment:
`plugin-item__count` seems to have associated styles outside this folder. Any side effects?
https://github.com/Automattic/wp-calypso/blob/a8021a658ce00e6d6795c2de8ddbb8c0eb69c081/client/my-sites/plugins/plugin-item/style.scss#L165-L202